src/jit/codegencommon.cpp

```diff
@@ -8760,10 +8760,10 @@ void CodeGen::genFnProlog()
     genZeroInitFrame(untrLclHi, untrLclLo, initReg, &initRegZeroed);
 
 #if FEATURE_EH_FUNCLETS
-
+#ifndef _TARGET_X86_
```

Comment:
I think we need it for x86 with EH funclets too. What was the reason for excluding it?